package.json

```diff
@@ -45,13 +45,13 @@
     "@angular/http": "4.3.6",
     "@angular/platform-browser": "4.3.6",
     "@blackbaud/skyux": "2.26.0",
-    "@blackbaud/skyux-builder": "1.24.0",
-    "@skyux-sdk/builder-plugin-skyux": "1.0.0-rc.0",
+    "@blackbaud/skyux-builder": "1.25.0",
+    "@skyux-sdk/builder-plugin-skyux": "1.0.0-rc.3",
     "@skyux/animations": "3.0.0-rc.0",
     "@skyux/assets": "3.0.0",
     "@skyux/core": "3.1.0",
     "@skyux/forms": "3.0.0-rc.0",
-    "@skyux/i18n": "3.0.1",
+    "@skyux/i18n": "3.2.0",
```

Comment:
This needs to be updated in the peerDependencies too. The versions for `forms`, `indicators`, and `popovers` also need to be updated (in both dev and peer)

Reply:
Just i18n is necessary

Reply:
Good catch